src/aura/BGE_BatchGiftEntryTab/BGE_BatchGiftEntryTab.cmp

```diff
@@ -0,0 +1,79 @@
+      notice, this list of conditions and the following disclaimer.
+    * Redistributions in binary form must reproduce the above copyright
+      notice, this list of conditions and the following disclaimer in the
+      documentation and/or other materials provided with the distribution.
+    * Neither the name of Salesforce.org nor the names of
+      its contributors may be used to endorse or promote products derived
+      from this software without specific prior written permission.
+
+    THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+    "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+    LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS
+    FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+    COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+    INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+    BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+    LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+    CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+    LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+    ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+    POSSIBILITY OF SUCH DAMAGE.
+-->
+<!--
+  @author Salesforce.org
+  @date 2018
+  @group Batch Gift Entry
+  @group-content ../../ApexDocContent/BatchDataImport.htm
+  @description Component used to view, edit, or create a Batch Template record.
+-->
+
+<aura:component implements="force:appHostable" controller="BGE_ConfigurationWizard_CTRL">
+
+    <!-- public attributes -->
+    <aura:attribute name="listView" type="Map" description="Namespace correct objectApiName and listName" access="public"
+                    default="{'isLoaded':false}"/>
+
+    <!-- events we handle -->
+    <aura:handler name="init" value="{!this}" action="{!c.doInit}" />
+
+    <!-- Start Markup -->
+    <lightning:overlayLibrary aura:id="overlayLib"/>
+
+    <!-- Banner -->
+    <header role="banner" class="header slds-p-bottom_medium">
+        <div class="slds-page-header">
+            <div class="slds-grid">
+                <div class="slds-col slds-has-flexi-truncate slds-align-middle">
+                    <h1>
+                        <span class="slds-page-header__title" title="{!$Label.c.bgeTabHeader}">{!$Label.c.bgeTabHeader}</span>
+                    </h1>
+                </div>
+                <div class="slds-col slds-no-flex slds-grid slds-align-top slds-p-bottom_xx-small headerButtonActions">
+                    <lightning:buttonGroup>
+                        <lightning:button aura:id="newBGEBatch" label="{!$Label.c.bgeNewBatch}" onclick="{!c.openNewBatchWizard}"/>
+                    </lightning:buttonGroup>
+                </div>
+            </div>
+        </div>
+    </header>
+
+    <!--listView does not rerender dynamically, so need aura:if to have it wait before loading-->
```

Comment:
What happens when the list view reference is invalid?

Reply:
Ideally I'd like to retrieve the list view name from Apex to make sure we have a match. As it is now, if the reference is invalid then this list view won't display.

Reply:
This seems like the kind of thing we would store in a custom setting so an Admin has visibility and control. Do you think we should do that at this stage, or iterate toward that later?

Reply:
I was thinking design attribute on the component. I think it's okay but not ideal for the list to simply not appear, but an error message might be better. We can move this into another story.